src/fillTank.test.js

```diff
@@ -1,11 +1,115 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared', () => {
+    expect(typeof (fillTank)).toBe('function');
+  });
+
+  it('If the amount is not given, then full tank is ordered.', () => {
```

Comment:
The test description mentions that if the amount is not given, a full tank should be ordered. However, the test calls `fillTank(customer, 1);` which specifies an amount. Consider modifying the test to omit the amount parameter to accurately test this scenario.